README.md

```diff
@@ -435,10 +435,36 @@ Assume this structure with the compressed asset as a sibling of the un-compresse
 └── index.html
 ```
 
-#### Disable serving
+#### Disable serving and CWD behavior
 
 If you would just like to use the reply decorator and not serve whole directories automatically, you can simply pass the option `{ serve: false }`. This will prevent the plugin from serving everything under `root`.
 
+When `serve: false` is passed:
+
+1. The plugin will not perform the usual directory existence check for the `root` option.
```

Comment:
too much information about internals

Reply:
devs dont want to know what the plugin does under the hood. here people want to know that sendFile will then send files from the current working directory.